### ISSUE TITLE
GrobidProperties, static methods vs getInstance

### ISSUE THREAD

**Issue by de-code** [kermitt2/grobid#440]
In the code, there is frequent use of config access like:

```java
GrobidProperties.getInstance().getDeLFTFilePath()
```

But `getDeLFTFilePath` in this case is a static method and it should be:

```java
GrobidProperties.getDeLFTFilePath()
```

However, `getInstance` actually ensures that the configuration is loaded in the first place.

**Comment by lfoppiano** [comment]
Yes, I'd say we should remove the static methods and use the `getInstance()` however I think there are probably several part where the statics are used.
